js/client-battle.js

```diff
@@ -1512,7 +1512,7 @@
 			if (this.gameType === 'battle' && this.room.battle && !this.room.battle.rated) {
 				buf += '<button type="button" name="replacePlayer" class="button">Replace player</button> ';
 			}
-			buf += '<button type="button" name="close" class="autofocus" class="button">Cancel</button></p></form>';
+			buf += '<button type="button" name="close" class="button">Cancel</button></p></form>';
```

Comment:
I'm pretty sure the intent was to add the `button` class without removing the `autofocus` class. Compare https://github.com/smogon/pokemon-showdown-client/blob/f9bc940eb5b2f2d55f7ea9b5c8c82e9ffe62c3c0/js/client.js#L2981
```suggestion
			buf += '<button type="button" name="close" class="autofocus button">Cancel</button></p></form>';
```
https://github.com/smogon/pokemon-showdown-client/blob/f9bc940eb5b2f2d55f7ea9b5c8c82e9ffe62c3c0/js/client.js#L2549 looks like it still needs fixing though.